### ISSUE TITLE
Wave background color is not updating

### ISSUE THREAD

**Issue by laurentiusandre** [alxrm/audiowave-progressbar#15]
I'm trying to change the color of the bar after audio is played. After the audio played, filled color is changed successfully but the background color is not changing to the new color. Here's a code snippet:

```
            waveView.setOnProgressListener(new OnProgressListener() {
                @Override
                public void onStartTracking(float v) {

                }

                @Override
                public void onStopTracking(float v) {

                }

                @Override
                public void onProgressChanged(float v, boolean b) {
                    if (v == 100F)
                        waveView.setWaveColor(ContextCompat.getColor(itemView.getContext(), R.color.subtitleGray));
                }
            });
```
XML:
```
        <rm.com.audiowave.AudioWaveView
            android:id="@+id/wave"
            android:layout_width="match_parent"
            android:layout_height="28dp"
            android:layout_marginEnd="20dp"
            app:animateExpansion="false"
            app:chunkWidth="3dp"
            app:chunkHeight="28dp"
            app:minChunkHeight="2dp"
            app:chunkSpacing="1dp"
            app:chunkRadius="2dp"
            app:waveColor="#ef4f35" />
```

Tested on Moto G with 7.1.2. Any fixes/workaround is appreciated!

**Comment by alxrm** [comment]
@laurentiusandre I guess I know(actually I just know) where the issue comes from, sorry for the slow response, maybe I'll fix it in the following days

That's a good catch, thanks!

**Comment by alxrm** [comment]
Fixed in [d779d42](https://github.com/alxrm/audiowave-progressbar/commit/d779d4243dff89acc5adc1000cbc247c3cdf886b)

**Comment by laurentiusandre** [comment]
Works! Thanks :+1:
